src/components/HomeViewComponents/HeaderSection.js

```diff
@@ -0,0 +1,54 @@
+import { Box, makeStyles } from "@material-ui/core";
+import React from "react";
+import Carousel from "react-material-ui-carousel";
+import HeaderItems from "./HeaderItems";
+
+const items = [
+  {
+    title: "Elastic Weekend Prizes worth 100K",
+    description:
+      "Learn about Building Search Engines, DevOps & Security, & Data Science 📈 ‼️ Prizes worth INR 1 lakh, including iPad, PS4, mobile phones, premium headphones, accessories, and more! 🤩",
+    img: "/assets/img/home.png",
+  },
+  {
+    title: "Elastic Weekend Prizes worth 100K",
+    description:
+      "Learn about Building Search Engines, DevOps & Security, & Data Science 📈 ‼️ Prizes worth INR 1 lakh, including iPad, PS4, mobile phones, premium headphones, accessories, and more! 🤩",
+
+    img: "/assets/img/home1.jpg",
+  },
+  {
+    title: "Elastic Weekend Prizes worth 100K",
+    description:
+      "Learn about Building Search Engines, DevOps & Security, & Data Science 📈 ‼️ Prizes worth INR 1 lakh, including iPad, PS4, mobile phones, premium headphones, accessories, and more! 🤩",
+
+    img: "/assets/img/learning.jpg",
+  },
+];
+function HeaderSection() {
+  const classes = useStyles();
+  return (
+    <Box className={classes.root}>
```

Comment:
Can we put a little border radius?
It will be much polished look.

Reply:
sure thing!